### DISCUSSION TITLE
all folders

### DISCUSSION THREAD

**Discussion by polipo1970** [Aries85/LightZone#343] [Ideas]
Is it possible to make on the browse section to select a folders and display all the other sub-folders?

**Comment by sfink16** [comment]
I'm not sure that I'm understanding.  You can drill down on the sub-folders by clicking on the higher level folder.  Perhsap you can provide a screenshot for better understanding.

**Reply by polipo1970** [reply]
Sorry, I misspoke. I meant if it is possible to see all the previews, of all the sub folders, for example if I have folder A,  that it includes B and C, I can see the previews if I open B or C, but if I go to A, I don't see the previews of B and C without opening. 
I hope I have been

**Reply by sfink16** [reply]
Not at the same time that I'm aware of.  You can go back and forth between folders by clicking the left or right arrow above the list of folders.  LZ is not the strongest functional raw convertor as it is very good at raw conversions, but lacks things like file management and screen layouts with user control.
